### ISSUE TITLE
Price Worker

### ISSUE THREAD

**Issue by vikiival** [kodadot/workers#77]
We need to store historical prices of Kusama for the indexers / frontend purposes

The simplest hack is to store the USD values (let's say price median) into the KV.

Where key would be date in preferred format (`YYYY-MM-DD`) and value would be json which would store price for tokens
- KSM
- BSX
- https://www.coingecko.com/en/coins/aleph-zero
- https://www.coingecko.com/en/coins/astar
- https://www.coingecko.com/en/coins/arbitrum maybe
- ETH?

ENDPOINTS:

GET `/get?date=""&tokens=[]`
POST `/getBatch`

and body should contain

```json
{
  "tokens": [] // list of tokens to be selected,
  "dates": [] // list of dates to be selected 
}
```

### Other stuff 

Be aware of limits

<img width="539" alt="Screenshot 2023-04-18 at 19 19 00" src="https://user-images.githubusercontent.com/22471030/232854970-d125e4c5-5c9e-449a-af96-1cd354ab4431.png">

We also need to add their logo in readme
- https://www.coingecko.com/en/branding

### Ref

- https://github.com/kodadot/nft-gallery/issues/5717

**Comment by yangwao** [comment]
> * Do you have any prefference for date format?

YYYY-MM-DD

> * Which other tokens would you fetch?

https://www.coingecko.com/en/coins/aleph-zero
https://www.coingecko.com/en/coins/astar
https://www.coingecko.com/en/coins/arbitrum maybe


> We also need to add their logo in readme

readme workers or readme in coingecko folder in workers 👀

**Comment by roiLeo** [comment]
Should be doable on indexer side, I remember seeing coingecko projects made with subsquid

**Comment by yangwao** [comment]
than that's probably different to GET, I was thinking they been running
separate indexer just with cg data

but I recall that it's dying as CG endpoint... timeouts?

cc @RaekwonIII

**Comment by vikiival** [comment]
TL;DR

![Screenshot 2023-04-19 at 15 45 18](https://user-images.githubusercontent.com/22471030/233094709-a74c2041-b750-4df6-9472-691ef699a2d5.png)

**Comment by vikiival** [comment]
cc @preschian there was an issue for that

great job 

Closed in #195
